<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorStatusBar</class>
 <widget class="QWidget" name="EditorStatusBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>35</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="fileSizeArea">
     <item>
      <widget class="QLabel" name="fileSizeLabel">
       <property name="text">
        <string>File Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fileSizeContent">
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="fileSizeAreaSep">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rowCountArea">
     <item>
      <widget class="QLabel" name="rowCountLabel">
       <property name="text">
        <string>Row Count:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rowCountContent">
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="rowCountAreaSep">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="charsetArea">
     <item>
      <widget class="QLabel" name="charsetLabel">
       <property name="text">
        <string>Charset:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="charsetContent">
       <property name="text">
        <string>Unknown</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="charsetSpecifyButton">
       <property name="text">
        <string>Specify</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="progressArea">
     <item>
      <widget class="QLabel" name="progressLabel">
       <property name="text">
        <string>Loading</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
